src/components/Routes/Categories.jsx

```diff
@@ -1,9 +1,25 @@
 import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { checkStatus } from '../../redux/categories/categorySlice';
 
 function Categories() {
+  const status = useSelector((state) => state.categories.categories);
+  const dispatch = useDispatch();
+
+  const statusCheck = () => {
+    dispatch(checkStatus());
+  };
+
   return (
     <div className="categories">
-      <button type="button" className="btn-check-status">Check Status</button>
+      <button
+        type="button"
+        className="btn-check-status"
+        onClick={statusCheck}
+      >
+        Check Status
+      </button>
+      <h4>{status}</h4>
     </div>
 
   );
```

Comment:
- [ ] Great effort working on your project, As stated in the project requirement, when the `Check Status` button is clicked on there should be a display text `Under Construction`
but current the button functionality is not working, when i clicked on it no text is displayed kindly fix this before resubmitting for a review.